recipes-browser/wpewebkit/wpewebkit_2.36.7.bb

```diff
@@ -13,11 +13,11 @@ DEPENDS += " libwpe"
 RCONFLICTS:${PN} = "libwpe (< 1.10) wpebackend-fdo (< 1.10)"
 
 SRC_URI:class-devupstream = "\
-    git://git.webkit.org/WebKit.git;branch=master \
+    git://github.com/WebKit/WebKit.git;protocol=https;branch=main \
     file://0002-libyuv-gcc-issue.patch \
 "
-# WPE 2.36.X branch was forked from the main branch in this commit
-SRCREV:class-devupstream = "b3f12a91b11b34b71aa4ec64c1806616b45bd877"
+# Can be used for branches for the next releases
```

Comment:
The `SRCREV:class-devupstream` should be only used to play with sources from repos and for development or evaluation purpose. The point of to keep the reference pointing to the specific commit from `main/master` where the release branch was forked as a quick reference.

In addition, a developer interested on the bleeding edge versions can simply including  the https://github.com/Igalia/meta-webkit/blob/main/conf/include/webkit-bleeding.inc  as part of the local.conf.

Reply:
Ack, I removed this part.